src/ilcdlib/epd/reader.py

```diff
@@ -89,6 +89,43 @@ def get_own_reference(self) -> IlcdReference | None:
         """Get the reference to this data set."""
         return IlcdReference(entity_type="processes", entity_id=self.get_uuid(), entity_version=self.get_version())
 
+    def get_lca_discussion(self, lang: LangDef) -> str | None:
```

Comment:
It isn't a great idea to perform a lot of concatenation operations in python or any other language when string is immutable type.
Instead use something like string builder pattern like this:

```
sb: list[str] = []
sb.append()
....
sb.append()
return "\n".join(sb)
```

Reply:
Done